feat(filters): add default visible filters for state and level

Piyush-r-bhaskar opened the pull request:
Closes https://github.com/kestra-io/kestra-ee/issues/5976

We now have a new prop `visibleByDefault` which if added with in any key in configurations file , will automatically add that filter to the filter list when the page loads as kinda `level in any`, even if no value to filter. Users can remove them, but they will reappear on page refresh.

https://github.com/user-attachments/assets/ef337411-1f8c-41cf-8793-e1a0cea8269a

Piyush-r-bhaskar committed:
feat(filters): add default visible filters for state and level

MilosPaunovic reviewed (approved):
Love this approach! 🚀

Piyush-r-bhaskar committed:
fix: missing translation warning from export csv

github-actions[bot] committed:
chore(core): localize to languages other than english (#13256)

Extended localization support by adding translations for multiple languages using English as the base. This enhances accessibility and usability for non-English-speaking users while keeping English as the source reference.

Co-authored-by: GitHub Action <actions@github.com>

MilosPaunovic committed:
Merge branch 'develop' into fix/add-default-state-and-level
